### PR TITLE
Added japanese localization files

### PR DESCRIPTION
**Pull request by harajune**
Hi. I created the Japanese localization files.
I would appreciate it if you could review it.

Thanks.

### CODE REVIEW

**Comment by sindresorhus** [comment]
Do you know anyone that could help review it?

File: Sources/KeyboardShortcuts/Localization/ja.lproj/Localizable.strings [hunk 1]
```diff
@@ -0,0 +1,5 @@
+"record_shortcut" = "キーを記録";
+"press_shortcut" = "キーを押してください";
+"keyboard_shortcut_used_by_menu_item" = "このショートカットは既に“%@”で使われており使えません。 ";
```
Comment (sindresorhus) [inline Sources/KeyboardShortcuts/Localization/ja.lproj/Localizable.strings:3]:
```suggestion
"keyboard_shortcut_used_by_menu_item" = "このショートカットは既に“%@”で使われており使えません。";
```

Should there really be a space at the end?

**Comment by fumiya-kume** [comment]
LGTM from Japanese native perspective.

File: Sources/KeyboardShortcuts/Localization/ja.lproj/Localizable.strings (hunk 1 above)
Comment (fumiya-kume) [reply Sources/KeyboardShortcuts/Localization/ja.lproj/Localizable.strings:3]: Maybe it's possible to remove it?

**Pull request merged**